common/ledger/util/leveldbhelper/leveldb_provider.go

```diff
@@ -149,6 +163,37 @@ func (h *DBHandle) Delete(key []byte, sync bool) error {
 	return h.db.Delete(constructLevelKey(h.dbName, key), sync)
 }
 
+// DeleteAll deletes all the keys that belong to the channel (dbName).
+func (h *DBHandle) DeleteAll() error {
+	iter := h.GetIterator(nil, nil)
+	defer iter.Release()
+
+	// This is common code shared by all the leveldb instances. Because each leveldb has its own key size pattern,
+	// each batch is limited by memory usage instead of number of keys. Once the batch memory usage reaches maxBatchSize,
+	// the batch will be committed.
+	numKeys := 0
+	batchSize := 0
+	batch := NewUpdateBatch()
+	for iter.Next() {
+		key := iter.Key()
```

Comment:
Just to take a note, since this function is inside leveldbhelper package, this can be made more efficient by directly operating at the level of leveldb iterator, keys, and batch. But since, this is expected to be a rare operation, I am fine with this as is.